src/main/kotlin/no/nav/syfo/personoppgavehendelse/kafka/PersonoppgavehendelseService.kt

```diff
@@ -18,12 +18,8 @@ class PersonoppgavehendelseService(
     private val database: DatabaseInterface,
 ) : KafkaConsumerService<KPersonoppgavehendelse> {
     override val pollDurationInMillis: Long = 100
-    val LPS_BISTAND_UBEHANDLET_TRUE = true
-    val LPS_BISTAND_UBEHANDLET_FALSE = false
-    val DIALOGMOTESVAR_UBEHANDLET_TRUE = true
-    val DIALOGMOTESVAR_UBEHANDLET_FALSE = false
-    val BEHANDLERDIALOG_SVAR_UBEHANDLET_TRUE = true
-    val BEHANDLERDIALOG_SVAR_UBEHANDLET_FALSE = false
+    private val isUbehandlet = true
+    private val isBehandlet = false
```

Comment:
Tanken her er å gjøre det mer lesbart i koden lenger ned her, i stedet for å ha variabler med true/false suffix i navnet. 
```kotlin
OversikthendelseType.DIALOGMOTESVAR_BEHANDLET ->
                   connection.updatePersonOversiktStatusDialogmotesvar(isBehandlet, personident)
```

Reply:
Men er det sånn at isUbehandler alltid skal være lik !isBehandlet? I så fall hadde det kanskje holdt med en variabel her?